### ISSUE TITLE
Sidebar disappears when scrolling code snippets in ReactJS documentation

### ISSUE THREAD

**Issue by ruturaj-gh** [reactjs/react.dev#5749]
![image](https://user-images.githubusercontent.com/83021083/226159867-3015aa43-9e48-4293-a4a3-20afff4639cf.png)
 here the sidebar for scrolling toward right side isn't visible but, if you scrolled it down to the snippet it is visible 
![image](https://user-images.githubusercontent.com/83021083/226159920-ed865b69-0351-4ddb-a9f9-72c165fbf5ac.png)
this makes it inconvenient while checking the code

**Comment by parthunified** [comment]
you can try using shift + scroll to move horizontally... I will try and fix this issue if no one else takes it up

**Comment by ruturaj-gh** [comment]
Yes , try to fix it . There might be a chance that they'll merge it in main.

**Comment by Mayank1170** [comment]
Hey @parthunified are you working on this ?
